c_tests/run.rs

```diff
@@ -150,7 +150,13 @@ fn run_suite(opt: &'static str) {
                 .starts_with("debug_")
     };
     #[cfg(cargo_profile = "debug")]
-    let filter: fn(&Path) -> bool = |p| p.extension().unwrap().to_str().unwrap() == "c";
+    let filter: fn(&Path) -> bool = |p| {
```

Comment:
Does this have anything to do with the long comments is this an unrelated refactor?

Reply:
It was crashing the unwrap because the `.clang_format` config file has no extension.

Reply:
Fair.